rundetection/rules/common_rules.py

```diff
@@ -14,3 +18,26 @@ class EnabledRule(Rule[bool]):
 
     def verify(self, job_request: JobRequest) -> None:
         job_request.will_reduce = self._value
+
+
+class NotAScatterFileError(Exception):
+    pass
+
+
+class CheckIfScatterSANS(Rule[bool]):
+    def verify(self, job_request: JobRequest) -> None:
+        if "_SANS/TRANS" not in job_request.experiment_title:
+            job_request.will_reduce = False
+            logger.error("Not a scatter run. Does not have _SANS/TRANS in the experiment title.")
+        # If it has empty or direct in the title assume it is a direct run file instead of a normal scatter.
+        if (
+            "empty" in job_request.experiment_title
+            or "EMPTY" in job_request.experiment_title
+            or "direct" in job_request.experiment_title
+            or "DIRECT" in job_request.experiment_title
+        ):
```

Comment:
```suggestion
        if (any(title in job_request.experiment_title.lower() for title in ["empty", "direct"]):
```

Reply:
Is this really better? Specifically is it clearer? Perhaps this instead?
```suggestion
        if (
            "empty" in job_request.experiment_title.lower()
            or "direct" in job_request.experiment_title.lower()
        ):
```

Reply:
I'm hearing what you are saying but the two are very different in terms of readability. When scanning through the code, your suggestion takes some time to understand, and mine is trivial to read at a glance and understand the function.

Reply:
That's fine